specification/devopsinfrastructure/Microsoft.DevOpsInfrastructure/examples/2024-04-04-preview/SubscriptionUsages_Usages.json

```diff
@@ -0,0 +1,36 @@
+{
+  "title": "SubscriptionUsages_Usages",
+  "operationId": "SubscriptionUsages_Usages",
+  "parameters": {
+    "subscriptionId": "a2e95d27-c161-4b61-bda4-11512c14c2c2",
+    "location": "eastus",
+    "api-version": "2024-04-04-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "unit": "Count",
+            "currentValue": 0,
+            "limit": 212,
+            "name": {
+              "value": "standardDADSv5Family",
+              "localizedValue": "Standard DADSv5 Family vCPUs (PME VMSS)"
+            }
+          },
```

Comment:
id, name and type are to be present as top level properties.
[https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#get-resource](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#get-resource)
 

I see id and name in the previous examples and not now.

Reply:
Added back id. As you can see in the contract, type doesn't exist, and name is a complex object: https://armwiki.azurewebsites.net/api_contracts/UsagesAPIContract.html?q=usages